presto-hive/src/test/resources/mock-krb5.conf

```diff
@@ -0,0 +1,8 @@
+# This file is CFEngine managed.  Contact #sysops on Slack or open a System Operations (SYSOPS) JIRA request for assistance.
+# filesource     ./sequences/policies/kerberos_client/templates/krb5.conf
+
+[libdefaults]
+default_realm = REALM1.COM
```

Comment:
- remove the configuration that doesn't matter. Eg. does `forwardable = true` matter? Or `udp_preference_limit` or `renew_lifetime`?
- give the file a meaningful name
- nit: indent